.prettierignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
Let's also remove `package.json`. `yarn` and `npm` write these in incompatible ways IIRC.

Reply:
✅